### DISCUSSION TITLE
blog/newsletters/2023-04-02-week-12-newsletter

### DISCUSSION THREAD

**Discussion by giscus** [redbrick/blog#27] [Blog]
# blog/newsletters/2023-04-02-week-12-newsletter

DCU's Networking Society.

https://redbrick.dcu.ie/blog/newsletters/2023-04-02-week-12-newsletter.html

**Comment by TrffnSec** [comment]
<img src=x>

**Comment by TrffnSec** [comment]
abcde

**Comment by redbrickCmt** [comment]
![image](https://github.com/user-attachments/assets/ae9813fa-1caf-4e07-b049-72cfa1f0e563)

**Reply by TrffnSec** [reply]
<img src="https://ih1.redbubble.net/image.893762967.8381/hj,750x-pad,750x1000,f8f8f8.u2.jpg">
